src/module.ts

```diff
@@ -148,7 +148,7 @@ export const enum FeatureFlags {
   MultiValue = 512 /* _BinaryenFeatureMultivalue */,
   GC = 1024 /* _BinaryenFeatureGC */,
   Memory64 = 2048 /* _BinaryenFeatureMemory64 */,
-  FunctionReferences = 4096 /* TODO_BinaryenFeatureTypedFunctionReferences */,
+  // TODO? GCNNLocals
```

Comment:
No, currenly Binaryen implemented 1a or 1b scheme for non-nullable locals. NN Local flags kept only for experimenting